### DISCUSSION TITLE
Schemata compatibility with dbt

### DISCUSSION THREAD

**Discussion by Ines70** [ananthdurai/schemata#79] [General]
hi,
I see that schemata works with a dbt provider but I don't understand how to integrate Schemata into a dbt project. Can you please help me  ☺☺ ?
 
Thanks

**Comment by ananthdurai** [comment]
Hi @Ines70, I actually just added the dbt backward compatibility rule with this PR https://github.com/ananthdurai/schemata/pull/80

I'm happy to jump on a call to explain how to incorporate your CI/ CD pipeline. This is the sample Github action to check backward compatibility for Protobuf.
https://github.com/ananthdurai/schemata/blob/main/.github/workflows/proto-schema-compatibility.yml
